common-template/src/test/java/helpers/SubscriptionWatchTestHelpers.java

```diff
@@ -0,0 +1,49 @@
+package helpers;
+
+import com.redhat.cloud.notifications.ingress.Action;
+import com.redhat.cloud.notifications.ingress.Context;
+import com.redhat.cloud.notifications.ingress.Event;
+import com.redhat.cloud.notifications.ingress.Metadata;
+import com.redhat.cloud.notifications.ingress.Payload;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static helpers.TestHelpers.DEFAULT_ORG_ID;
+
+public class SubscriptionWatchTestHelpers {
+
+    public static Action createSubscriptionWatchAction() {
+        Action emailActionMessage = new Action();
+        emailActionMessage.setBundle("subscription-services");
+        emailActionMessage.setApplication("subscription-watch");
+        emailActionMessage.setTimestamp(LocalDateTime.of(2020, 10, 3, 15, 22, 13, 25));
+        emailActionMessage.setEventType("usage-threshold-exceeded");
+        emailActionMessage.setRecipients(List.of());
+
+        emailActionMessage.setContext(
+                new Context.ContextBuilder()
+                        .withAdditionalProperty("org_id", "123456789")
```

Comment:
The org Id is already present at the higher level, as you defined it at the end of this method  (`emailActionMessage.setOrgId(DEFAULT_ORG_ID);`)